pkg/process/util/orchestrator/cache.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This code does not seem particularly related to process, not sure `pkg/process/util/` is the best place for it.

Reply:
Same as above we decided to put it here and opened a follow up story to move some packages out to its own packages. Like 'pkg/orchestrator'
👍